backend/src/db.js

```diff
@@ -2,9 +2,12 @@ import Sequelize from 'sequelize';
 
 import User from './models/user.model';
 
+const path = require('path');
+const os = require('os');
```

Comment:
I'm sorry! This code works perfectly, but I forgot I used es6 imports in code. Could you replace these 2 lines with

```javascript
import path from 'path';
import os from 'os';
```

My bad, gave you some mistaken tips above.